### PR TITLE
Create Vault Options:  CoVault & Yield Generating CoVault

### PR DESCRIPTION
**Pull request by wingfeatherwave**
Screen Shot 2022-08-19 at 4 28 43 PM

References: https://www.notion.so/coordinape/Create-CoVault-Change-Hierarchy-a8a69e49c1e14d6889d7a0176e73d093

Big thanks to @crabsinger for all the pairing help!

@levity @topocount please take a look at the hard coded USDC and DAI addresses (CreateForm.tsx:Line 19-20).  Is this an okay way to do things for now?  Is there another, better way?

### CODE REVIEW

**Comment by levity** [comment]
> Is there another, better way?

Yes; use [`contracts.getTokenAddress`](https://github.com/coordinape/coordinape/blob/main/src/lib/vaults/contracts.ts#L117)

File: src/pages/VaultsPage/CreateForm.tsx [hunk 1]
```diff
@@ -155,63 +189,113 @@ export const CreateForm = ({
...
-        {...customAddressField}
-      />
+      </Panel>
+      <HR />
+      <Panel
+        invertForm={activeVaultPanel === 'yearn'}
+        nested={activeVaultPanel !== 'yearn'}
+        css={{ gap: '$md' }}
+      >
+        <Text h3>Yield Generating CoVault</Text>
```
Comment (levity) [inline src/pages/VaultsPage/CreateForm.tsx:268]:
```suggestion
        <Text h3>Yield-Generating CoVault</Text>
```

**Review by levity — APPROVED** [review]
> > Is there another, better way?
> 
> Yes; use [`contracts.getTokenAddress`](https://github.com/coordinape/coordinape/blob/main/src/lib/vaults/contracts.ts#L117)

Just made this change and a few other small ones. The code looks good otherwise; nice work with toggling the panel backgrounds.

**Comment by wingfeatherwave** [comment]
Thank you @levity for the changes!

**Pull request merged**